daemon/mgr/network.go

```diff
@@ -149,26 +150,86 @@ func (nm *NetworkManager) List(ctx context.Context, labels map[string]string) ([
 	return net, nil
 }
 
-// Get returns the information of network that specified name/id.
-func (nm *NetworkManager) Get(ctx context.Context, name string) (*types.Network, error) {
+// GetNetworkByName returns the information of network that specified name.
+func (nm *NetworkManager) GetNetworkByName(name string) (*types.Network, error) {
 	n, err := nm.controller.NetworkByName(name)
-	if err != nil {
-		if err == libnetwork.ErrNoSuchNetwork(name) {
-			return nil, errors.Wrap(errtypes.ErrNotfound, err.Error())
-		}
-		return nil, err
-	}
-
-	if n != nil {
+	if err == nil {
```

Comment:
If the logic is correct, I should suggest to use return fast to avoid the large ident. Like:
```
if err != nil {
    return nil, err
}
return &types.Network{
     ......
}
```

Reply:
@allencloud Thanks a lot for your reviews.
I have modified the code according to your suggestions.